app/frontend/wallet/helpers/tokenFormater.ts

```diff
@@ -35,9 +35,11 @@ export const aggregateTokenBundles = (tokenBundle: TokenBundle[]): TokenBundle =
 
 export const groupTokenBundleByPolicyId = (
   tokenBundle: TokenBundle
-): {[policyId: string]: TokenBundle} => {
+): {[policyId: string]: Omit<Token, 'policyId'>[]} => {
   return _(tokenBundle)
+    .orderBy(['policyId', 'assetName'], ['asc', 'asc'])
```

Comment:
this may still not work properly for assetNames - the cbor canonical ordering instructs to put shorter keys first:

```
*  If two keys have different lengths, the shorter one sorts earlier;
*  If two keys have the same length, the one with the lower value in (byte-wise) lexical order sorts earlier.
```


source: https://tools.ietf.org/html/rfc7049#section-3.9